scss/_patterns_chip.scss

```diff
@@ -0,0 +1,75 @@
+@import 'settings';
+
+@mixin vf-p-chip {
+  $chip-line-height: map-get($line-heights, default-text);
+  $chip-vertical-padding: 0;
+  $chip-height: $chip-line-height + 2 * $chip-vertical-padding;
+  $chip-margin-top: map-get($nudges, nudge--p) - $chip-vertical-padding;
+
+  .p-chip {
+    @extend %small-text;
+
+    background-color: rgba($color-x-dark, 0.05);
+    border-radius: $chip-height * 0.5;
+    display: inline-block;
+    line-height: $chip-line-height;
+    margin: $chip-margin-top $sph-inner--small 0 0;
+    padding: $chip-vertical-padding $sph-inner;
+    user-select: none;
+    white-space: nowrap;
+
+    &:hover {
+      background-color: rgba($color-x-dark, 0.08);
+    }
+
+    &.is-selected {
+      background-color: rgba($color-x-dark, 0.15);
+
+      &:hover {
+        background-color: rgba($color-x-dark, 0.18);
+      }
+    }
+
+    &:active {
+      background-color: rgba($color-x-dark, 0.15);
+    }
+
+    .p-chip__lead {
+      @extend %x-small-text;
+
+      display: inline;
+      line-height: $chip-line-height;
+      text-transform: uppercase;
+    }
+
+    .p-chip__value {
+      @extend %small-text;
+
+      display: inline;
+      line-height: $chip-line-height;
+      position: relative;
```

Comment:
I don't think you need display: relative here as your not changing the position.